recipes/matchms/meta.yaml

```diff
@@ -27,8 +27,8 @@ requirements:
     - pyteomics >=4.2
     - python
     - pyyaml
```

Comment:
> removed a no longer used dependency (pyyaml)

They `pyyaml` has not been removed, `requests` were.

Reply:
Thanks for spotting this! I changed it